collectors/cronjob_test.go

```diff
@@ -49,6 +49,8 @@ func TestCronJobCollector(t *testing.T) {
 	const metadata = `
 		# HELP kube_cronjob_info Info about cronjob.
 		# TYPE kube_cronjob_info gauge
+		# HELP kube_cronjob_created Unix creation timestamp
```

Comment:
@julia-stripe should add a `kube_cronjob_created` metric here just like what you added for [`daemonset_test.go`](https://github.com/kubernetes/kube-state-metrics/pull/182/files#diff-edd133937b9a4e7fe14e47765a166c71R73)